projects/aca-content/folder-rules/src/rule-details/conditions/rule-simple-condition.ui-component.ts

```diff
@@ -102,6 +127,11 @@ export class RuleSimpleConditionUiComponent implements ControlValueAccessor, OnD
   get isComparatorHidden(): boolean {
     return comparatorHiddenForConditionFieldType.includes(this.selectedField?.type);
   }
+
+  get fieldControl(): AbstractControl {
```

Comment:
Do we need that getter at all? It is used only in single place so it does not give us too much value as we can write that code directly in place of usage.

Reply:
Removed getter. Code now uses this.form.get('field') directly